### DISCUSSION TITLE
Taxonomies: Custom metabox UI

### DISCUSSION THREAD

**Discussion by JoryHogeveen** [pods-framework/pods#6702] [Ideas]
https://developer.wordpress.org/reference/functions/register_taxonomy/
See `meta_box_cb` and `meta_box_sanitize_cb`.

Currently we only support the default display types for taxonomies.
I've seen requests/questions about this in multiple occasions where people want to allow only one taxonomy or just a different UI type.

With the pick field UI types and the DFV add-new options we could provide various UI types, compatible with the blocks editor, to select/manage taxonomies attached to posts.

**Comment by jimtrue** [comment]
As long as this gets us away from people using relationships for taxonomy, I'd suggest this as a good thing.
